Inc/HALAL/Services/Encoder/Encoder.hpp

```diff
@@ -27,43 +27,43 @@ class Encoder {
 	 *
 	 * @retval optional<uint8_t> Id of the service or empty if the pin pair is not valid
 	 */
-	static optional<uint8_t> register_encoder(Pin& pin1, Pin& pin2);
+	static optional<uint8_t> register_(Pin& pin1, Pin& pin2);
```

Comment:
Se te ha colado una barra baja. 
```suggestion
	static optional<uint8_t> register(Pin& pin1, Pin& pin2);
```